xsuite/src/world/fsworld.test.ts

```diff
@@ -644,6 +668,15 @@ test.concurrent("FSWallet.getAccountBalance", async () => {
   expect(await wallet.getAccountBalance()).toEqual(10n ** 18n);
 });
 
+test.concurrent("FSWallet.getAccountValue", async () => {
+  using world = await FSWorld.start();
+  const { wallet } = await createAccounts(world);
```

Comment:
Let's not use `createAccounts` and do something simple. We don't need to create sophisticated keys as it should not change the functionality.